backend/deploy_prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+# Check for a file only in the root dir
+if [[ ! -f playwright.config.js ]]; then
+  echo "You must run this script from the root directory of the repo"
+  exit
+fi
+
+read -p "Do you want to deploy to prod? (y/n)?" CONFIRM
+if [ "$CONFIRM" != "y" ]; then
+  exit
+fi
+
+# Build the app
+npm ci
```

Comment:
ci, not i, to make sure we use package-lock. Following exactly what's in the working GH workflows